packages/flutter/test/widgets/navigator_replacement_test.dart

```diff
@@ -39,42 +42,202 @@ void main() {
     expect(find.text('home'), findsOneWidget);
   });
 
-  testWidgets('pushAndRemoveUntil', (WidgetTester tester) async {
-    await tester.pumpWidget(MaterialApp(
+  group('pushAndRemoveUntil', () {
+
+    final TestObserver observer = TestObserver();
+    final Widget myApp = MaterialApp(
       home: const Material(child: Text('home')),
       routes: <String, WidgetBuilder>{
         '/a': (BuildContext context) => const Material(child: Text('a')),
         '/b': (BuildContext context) => const Material(child: Text('b')),
+        '/c': (BuildContext context) => const Material(child: Text('c')),
```

Comment:
Is `/c` actually used anywhere? (If we don't have it, it would probably be good to have a test that removes more than just one route...)